ONSdigital/eq-survey-runner
# Eq 14 add test setup

**What**

Reimplements the healthtest and hello world page on new project layout. Modifies the test accordingly

**How to test**

_Pre-test_

a) Install the requirements for tests: `requirements_for_test.txt` 

1) Run the app using `scripts/run_app.sh`
2) Verify the Hello World page at the root of the web application using the browser
3) Click the healthcheck link and view the JSON
4) Run all the tests using `scripts/run_tests.sh`

**Who can test**

Anybody except @weapdiv-david
